drivers/sensor/openrocket/openrocket_sensors.c

```diff
@@ -113,8 +119,8 @@ static struct or_data_t pad_packet = {
     .yaw = 0,
 #endif
 #ifdef CONFIG_OPENROCKET_BAROMETER
-    .pressure = or_packets[0].pressure,
-    .temperature = or_packets[0].temperature,
+    .pressure = 0,
```

Comment:
The haters (the compiler) tell me `initializer element is not constant` :roll_eyes: